packages/logger/src/logger-context.ts

```diff
@@ -18,7 +20,13 @@ export class LoggerContext {
   }
 
   async wrap(fn: (...args: unknown[]) => unknown, initialState = {}) {
-    return this.als.run(initialState, fn);
+    return this.als.run(
+      {
+        ...initialState,
+        project_name: this.projectName,
```

Comment:
also add `service`. In otel and datadog `service` is the right name. `project_name` is extra for vercel

Reply:
ah good, thats fine